src/gowrapper/ckks/params.go

```diff
@@ -174,6 +193,12 @@ func lattigo_sigma(paramHandle Handle6) float64 {
 	return params.Sigma()
 }
 
+//export lattigo_beta
+func lattigo_beta(paramHandle Handle6) int {
```

Comment:
I tried to be conservative and explicit about cross-language type conversions by converting all Go `int` types to `uint64`. Could you make this like `lattigo_logN`?